### PR TITLE
Support STORE_ATTR and STORE_SUBSCR

### PR DESCRIPTION
**Pull request by pwwang**
As discussed in #36.

### CODE REVIEW

**Comment by alexmojaki** [comment]
@pwwang did you see https://github.com/pwwang/executing/pull/1?

**Comment by pwwang** [comment]
> @pwwang did you see https://github.com/pwwang/executing/pull/1?

Ah nope. will take a look.

File: tests/test_main.py [hunk 1]
```diff
@@ -545,6 +566,7 @@ def check_code(self, code, nodes, decorators, check_names):
                 (
                     'BINARY_', 'UNARY_', 'LOAD_ATTR', 'LOAD_METHOD', 'LOOKUP_METHOD',
                     'SLICE+', 'COMPARE_OP', 'CALL_', 'IS_OP', 'CONTAINS_OP',
+                    'STORE_SUBSCR', 'STORE_ATTR',
```
Comment (alexmojaki) [inline tests/test_main.py:569]:
```suggestion
                    'STORE_SUBSCR', 'STORE_ATTR', 'STORE_SLICE',
```

**Comment by alexmojaki** [comment]
https://github.com/pwwang/executing/pull/2

Btw I don't seem to be able to push directly to your fork, hence these PRs. If you made a branch on your fork it might work. Or you could make a branch here, I added you as a collaborator.

**Comment by pwwang** [comment]
Is the branch `pwwang/master` in this repo a link or a copy to my fork? Should I just modify this branch?

**Comment by alexmojaki** [comment]
It's a copy. But the second PR above is using a new branch anyway.

**Comment by pwwang** [comment]
I have merged your commits to my fork anyway even though it still fails with python3.7. 
Do you mind resetting branch `pwwang/master` to the HEAD of my fork? Otherwise, I can create a new branch to the HEAD of my fork here.

**Comment by pwwang** [comment]
Let's close this and use #38 instead.

**Pull request closed without merging**